src/GafferScene/Parent.cpp

```diff
@@ -184,7 +184,8 @@ void Parent::hash( const Gaffer::ValuePlug *output, const Gaffer::Context *conte
 
 	if( output == mappingPlug() )
 	{
-		ScenePlug::PathScope scope( context, ScenePath() );
+		const ScenePath root;
```

Comment:
In Group you declared `g_root` and used that, which I liked.

Reply:
Addressed in e15eea3.